doc/api/api.yaml

```
@@ -2854,6 +2840,423 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  # ms Space
+  /space/info:
+    get:
+      summary: Get space parameters info
+      operationId: handler-info-get
+      tags:
+        - main
+      parameters:
+        - $ref: '#/components/parameters/XUserHeader'
+      responses:
+        '200':
+          description: Space parameters
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SpaceParams"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /space/invitation:
+    get:
+      summary: Get invitations list
+      operationId: handler-invitation-get
+      tags:
+        - main
```

Нужно тегами разбить ручки по микросервисам. 
```suggestion
        - space
```

---

Готово.